dotcom-rendering/src/web/components/LightboxPicture.tsx

```diff
@@ -0,0 +1,122 @@
+import { css } from '@emotion/react';
+import { breakpoints } from '@guardian/source-foundations';
+import React from 'react';
+import type { ImageWidthType } from './Picture';
```

Comment:
Maybe we could have a `picture` file for these sorts of shared types? That seems to be the pattern